<launch>
    <arg name="dev" default="/dev/ttyACM0"/>
    <arg name="baud" default="115200"/>

    <arg name="camera_topic" default="/usb_cam/image_raw/compressed"/>
    <arg name="jpeg_quality_level" default="40"/>

    <node pkg="nordic_node" type="camera_test.py" name="nordic_send" output="screen">
        <param name="~dev" value="$(arg dev)"/>
        <param name="~baud" value="$(arg baud)"/>
        <param name="~camera_topic" value="$(arg camera_topic)"/>
        <param name="~jpeg_quality_level" value="$(arg jpeg_quality_level)"/>
    </node>
</launch>